libraries/abstractions/ble_hal/test/ble_test_scipts/startTests_afqp.py

```diff
@@ -32,6 +32,7 @@
 from testClass import runTest
 from bleAdapter import bleAdapter
 
+ENABLE_TC_AFQP_READ_WRITE_LONG = 1
```

Comment:
Is this only used as a boolean? Might lead to some confusion later when using an int as a bool

Reply:
Right, the config was left as an integer, so as to match the corresponding config on the DUT side. Eventually this needs to be moved to a config shared by both DUT and RPI python tests.

Reply:
Oh I see, thanks !